src/components/shapes/movieCardShape.js

```diff
@@ -0,0 +1,10 @@
+import PropTypes from 'prop-types';
+
+const movieCardShape = PropTypes.shape({
```

Comment:
шейпы с большой буквы
```suggestion
const MovieCardShape = PropTypes.shape({
```